compiler-rt/lib/builtins/cpu_model/x86.c

```diff
@@ -646,10 +657,102 @@ static const char *getIntelProcessorTypeAndSubtype(unsigned Family,
       *Type = INTEL_KNM;
       break;
 
-    default: // Unknown family 6 CPU.
+    default: // Unknown family 6 CPU, try to guess.
```

Comment:
I recall I saw discussion somewhere says it's not proved returning a guessed CPU is better than unknown one.

Reply:
Do you have a link? Should we remove this then? This just unifies the implementations, so if we shouldn't guess, then we should remove it from the LLVM implementation.

More than happy to put up a patch for that if that's what is desired.

Reply:
No, I can't find it. I'm even not quite sure of it. I think we can put both way in an RFC and see others' opinion. Do you know the behavior of libgcc? It would be a good reference.